### ISSUE TITLE
Go to returns wrong date

### ISSUE THREAD

**Issue by toreskogp** [spencermountain/spacetime#122]
I am getting a strange result when using goto, where the date ends up being the date before it is supposed to be.

I am only getting this bug between 0:00 and 1:59 and my timezone is Europe/Berlin (UTC+2).

```
const spacetime = require('spacetime');
console.log(spacetime('2019-04-05T06:00:00Z').goto('Europe/Berlin').format('iso'));
// 2019-04-04T08:00:00.000+02:00
```

**Comment by spencermountain** [comment]
ah, right. so the hours are correct, but the date is flipping back.
this looks like a good bug. I'll check it out, 
thanks

**Comment by spencermountain** [comment]
this is now fixed on `5.7.0`
thanks!

**Comment by toreskogp** [comment]
Thanks!
